gov_uk_dashboards/formatting/human_readable.py

```diff
@@ -0,0 +1,23 @@
+"""Functions for converting values to human readable format"""
+
+
+def format_as_human_readable(value_to_format: float, *, prefix: str = "") -> str:
+    """Format a number as a human readable string
+    Appends bn, m, k as appropriate
+    Rounds number to 3dp
+
+    Args:
+        value_to_format(float): number to format
+        prefix(str,optional): prefix to append to the start
+
+    Returns:
+        str: formatted number
+    """
+
+    if value_to_format >= 1_000_000_000:
+        return f"{prefix}{value_to_format/1_000_000_000:g}bn"
+    if value_to_format >= 1_000_000:
+        return f"{prefix}{value_to_format/1_000_000:g}m"
+    if value_to_format >= 1_000:
+        return f"{prefix}{value_to_format/1_000:g}k"
+    return f"{prefix}{value_to_format:g}"
```

Comment:
What does the :g do?

Reply:
It applied a 'general format' to numbers - for us, that means rounding to 3dp if necessary and removing any training zeroes.